src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -194,18 +194,23 @@ public Func<EndpointInfo> OtlpServiceHttpEndPointAccessor
 
         _logger = GetLogger();
 
-        // this needs to be explicitly enumerated for each supported language
-        // our language list comes from https://github.com/dotnet/arcade/blob/89008f339a79931cc49c739e9dbc1a27c608b379/src/Microsoft.DotNet.XliffTasks/build/Microsoft.DotNet.XliffTasks.props#L22
-        var supportedLanguages = new[]
+        // our localization list comes from https://github.com/dotnet/arcade/blob/89008f339a79931cc49c739e9dbc1a27c608b379/src/Microsoft.DotNet.XliffTasks/build/Microsoft.DotNet.XliffTasks.props#L22
+        var localizedLanguages = new[]
         {
             "en", "cs", "de", "es", "fr", "it", "ja", "ko", "pl", "pt-BR", "ru", "tr", "zh-Hans", "zh-Hant", // Standard cultures for compliance.
-            "zh-CN", // Non-standard culture but it is the default in many Chinese browsers. Adding zh-CN allows OS culture customization to flow through the dashboard.
-            "en-GB", // Support UK DateTime formatting (24-hour clock, dd/MM/yyyy)
-        };
+        }.ToHashSet();
+
+        var supportedCultures = CultureInfo.GetCultures(CultureTypes.AllCultures)
+            .Where(culture => localizedLanguages.Contains(culture.Name) || localizedLanguages.Contains(culture.TwoLetterISOLanguageName))
+            .Select(culture => culture.Name)
+            .ToList();
```

Comment:
Split out into a method and write a unit test to check this works correctly. Double check a few, popular, expected cultures are in the supported cultures.

Reply:
e.g. all the values in localizedLanguages, en-GB, en-US, zh-CN, de-DE, fr-FR, etc.

Reply:
done